### ISSUE TITLE
PyPI project name

### ISSUE THREAD

**Issue by Natooz** [obfusk/m#3]
Dear Fay @obfusk,

I recently tried to contact you by email to discuss on the possibility for you to free the `mmm` project name on [PyPI](https://pypi.org/project/mmm/).
With no response, I am opening an issue here with the hope to get your attention.

Here is the content of the original email with the details:

> Dear Fay,
> 
> I hope this email finds you well.
> I am writing to you as I am interested in the “mmm” package name on PyPi that you currently own.
> 
> I am a postdoctoral researcher working at the Metacreation Lab at Simon Fraser University. I currently work on making an open-source version of the “MMM” project of the lab. You can find the original preprint here: https://arxiv.org/abs/2008.06048
> It stands for Multitrack Music Machine and is a deep learning model allowing to help musicians to create music, by generating portions of tracks in a co-creative way. We are currently working on an updated version (more performant and more efficient) that we plan to release on Hugging Face, and on PyPi to provide an easy to use package as using the model requires a bit of data manipulation in order to create “appropriate” prompts. :)
> 
> Naturally, we would love to use the “mmm” handle on PyPi as it is short and is the name of the project/model. I saw that your mmm project hasn’t been updated since 2018. Consequently, may I ask you if you would be willing to rename it on PyPi to let us use it? Or is there something we can do to make you accept?
> 
> Best,
> Nathan

Please let me know if I can do anything.

Nathan

**Comment by obfusk** [comment]
I did not receive your email. I don't know why. And I'm sorry, but no, I understand why you'd like the package name but:

- `mmm` is still used and maintained. It simply hasn't needed any updates since 2018. There have been no bugs to fix. No need for a new release. It "just works" :)
- Reusing names _that have actually been used, not just "squatted"_ is bad security. No offence, but I don't know you. Someone wanting to install or update my package would likely end up installing yours. And I cannot guarantee that would be safe for them.
- I may very well release a new version of `mmm` in the future if something needs fixing. A rename would break upgrading.

So I'm sorry, I would have considered using a different package name had I been able to anticipate this, but I have no intention of renaming.
